requirements.txt

```diff
@@ -10,10 +10,12 @@ mistune>=0.8.4
 numpy>=1.20.3,<2.0.0; python_version == "3.8"
 numpy>=1.21.6,<2.0.0; python_version == "3.9"
 numpy>=1.22.4,<2.0.0; python_version >= "3.10"
+numpy>=1.26.0,<2.0.0; python_version >= "3.12"
 packaging
 pandas>=1.1.0; python_version <= "3.8"
 pandas>=1.1.3; python_version == "3.9"
 pandas>=1.3.0; python_version >= "3.10"
+pandas>=2.2.0; python_version >= "3.12"
```

Comment:
Updated these minimum versions to match what pandas + numpy say is the earliest version that supports `3.12`